test/module/shared_model/validators/field_validator_test.cpp

```diff
@@ -76,7 +64,7 @@ class FieldValidatorTest : public ValidatorsTest {
 
  public:
   FieldValidatorTest() {
-    for (const auto &field : {"public_key", "main_pubkey", "pubkey"}) {
+    for (const auto &field : {"public_key", "public_key", "pubkey"}) {
```

Comment:
One "public_key" should be enough here, because it populates a collection. Is "pubkey" used anywhere? It can be removed if it is unused.